drivers/1wire/1wire.c

```diff
@@ -110,11 +110,9 @@ static inline uint32_t onewire_leuint32(uint32_t x)
  *
  ****************************************************************************/
 
-static inline void onewire_sem_init(FAR struct onewire_sem_s *sem)
+static inline void onewire_sem_init(FAR rmutex_t *lock)
```

Comment:
maybe can replace `onewire_sem_init` with `nxrmutex_init` in all the places? Or do a define